generators/cassandra/generator.mjs

```diff
@@ -32,22 +32,36 @@ export default class CassandraGenerator extends BaseApplicationGenerator {
     }
   }
 
+  get configuringEachEntity() {
+    return this.asConfiguringEachEntityTaskGroup({
+      checkEntities({ entityConfig }) {
+        if (entityConfig.pagination && entityConfig.pagination !== 'no') {
```

Comment:
Can't derived variable be loaded before configuring? Would avoid the '!== 'no''

Reply:
No, but doing validation at a later step is something to consider.
Priorities exclusively for validations is something to consider.
We should open an issue for this.